src/containers/paper-canvas.jsx

```diff
@@ -121,6 +122,7 @@ class PaperCanvas extends React.Component {
         const img = new Image();
         img.onload = () => {
             const raster = new paper.Raster(img);
+            raster.remove();
```

Comment:
What does this do?

Reply:
I was seeing an issue where the raster was appearing for a split second at 0, 0 and I thought this might help. I'm not sure if it fixed it... Without this line, the raster would get removed a bit later during paper.project.activeLayer.removeChildren();